From 0b1a11f497619c9628f332fa1807496828d76124 Mon Sep 17 00:00:00 2001
From: shamim ali <ashamim@ami.com>
Date: Sun, 23 Nov 2025 11:01:23 -0500
Subject: [PATCH] Added sel delete status property This property will update
 the delete status to true after deleting the last entry using delete sel
 entry command, the status will be retrived by the LogStatus sensor to not add
 clear sel event during this time.

Signed-off-by: shamim ali <ashamim@ami.com>
---
 include/defaults.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/defaults.hpp b/include/defaults.hpp
index 668df4c..c543f4f 100644
--- a/include/defaults.hpp
+++ b/include/defaults.hpp
@@ -490,6 +490,7 @@ setting->addProperty("RetryIntervalMS", static_cast<uint8_t>(20));
 
     setting->addProperty("lastEntryId", static_cast<uint16_t>(0));
     setting->addProperty("ipmiEntryCount", static_cast<uint16_t>(0));
+    setting->addProperty("selDelStatus", false);
 
 
     setting = &settings.emplace_back(objectServer,
-- 
2.34.1

